### PR TITLE
Style cleanup for `models/jules/R/*.R` scripts

### LINKED ISSUE #1051: Comparison used in place of assignment in write.config.JULES.R:285
Similar to issue #1049 - this is clearly wrong.

```R
      if(var == "emis_v") { ## Surface emissivity. BETY: per mil
        names(pft)[v] == "emis_pft_io"  
        pft[v] = pft[v]/1000
      }
```

### PR DESCRIPTION
**Pull request by bpbond**
Ran through formatR::tidy_dir with arrow=TRUE and indent=2; made sure
all `if` blocks have braces. Tried to make sure formatR didn’t make
things too jaggy.

@mdietze  note lines 194-206 in `write.config.JULES.R` - I have chained
these ifs together and given them a failure `stop()` - is this OK?

Also fixes #1051.

### CODE REVIEW

File: models/jules/R/write.config.JULES.R [hunk 1]
```diff
@@ -24,316 +24,335 @@
...
+      ## for the leaf reflection coefficient for VIS b_wl_io ## Allometric exponent relating the target
+      ## woody biomass to the leaf area index.  catch0_io ## Minimum canopy capacity (kg m-2).
+      ## dcatch_dlai_io ## Rate of change of canopy capacity with LAI (kg m-2).  Canopy capacity is
+      ## calculated as catch0 + dcatch_dlai*lai dgl_dm_io ## Rate of change of leaf turnover rate with
+      ## moisture availability dgl_dt_io ## Rate of change of leaf turnover rate with temperature (K-1)
+      ## dqcrit_io ## Critical humidity deficit (kg H2O per kg air).  dz0v_dh_io ## Rate of change of
+      ## vegetation roughness length for momentum with height.  Roughness length is calculated as dz0v_dh
+      ## * canht_ft eta_sl_io ## Live stemwood coefficient (kg C/m/LAI) fd_io ## Scale factor for dark
+      ## respiration **** look up equation fsmc_of_io ## Moisture availability below which leaves are
+      ## dropped f0_io ## CI / CA for DQ = 0 Minimum turnover rate for leaves (/360days); BETY: year-1
```
Comment (mdietze) [inline models/jules/R/write.config.JULES.R]: Ok so there's a huge chunk of comments here and above that tidy just mangled. Previously, there was one variable per line for the parameters that we haven't yet matched between JULES and PEcAn. Now they're just text blobs. Would be very helpful to revert these blocks.

File: models/jules/R/write.config.JULES.R [hunk 2]
```diff
@@ -24,316 +24,335 @@
...
+      }
+      ## kpar_io ## PAR Extinction coefficient (m2 leaf / m2 ground) neff_io ## Scale factor relating
+      ## Vcmax with leaf nitrogen concentration nl0_io ## Top leaf nitrogen concentration (kg N/kg C)
+      ## nr_nl_io ## Ratio of root nitrogen concentration to leaf nitrogen concentration calcuate this
+      ## from leaf and root N as well as leaf and root C:N ns_nl_io ## Ratio of stem nitrogen
+      ## concentration to leaf nitrogen concentration omega_io ## Leaf scattering coefficient for PAR
+      ## omegau_io ## Upper limit for the leaf scattering coefficient for PAR omegal_io ## Lower limit
+      ## for the leaf scattering coefficient for PAR omnir_io ## Leaf scattering coefficient for NIR
+      ## omniru_io ## Upper limit for the leaf scattering coefficient for NIR omnirl_io ## Lower limit
+      ## for the leaf scattering coefficient for NIR
```
Comment (mdietze) [inline models/jules/R/write.config.JULES.R]: here also

File: models/jules/R/write.config.JULES.R [hunk 3]
```diff
@@ -24,316 +24,335 @@
...
-      if(var == "leafN") { ## Top leaf nitrogen content per unit mass (kgN kgLeaf-1); BETY: percent 
-        names(pft)[v] = "nmass_io"
-        pft[v] = pft[v]/100
+      ## fl_o3_ct_io ## Critical flux of O3 to vegetation (mmol m-2 s-1) dfp_dcuo_io ## Fractional
+      ## reduction of photosynthesis with the cumulative uptake of O3 by leaves (mmol m-2) ief_io ##
+      ## Isoprene Emission Factor (μg g-1 h-1) tef_io ## Monoterpene Emission Factor (μg g-1 h-1) mef_io
+      ## ## Methanol Emission Factor (μg g-1 h-1) aef_io ## Acetone Emission Factor (μg g-1 h-1) ci_st_io
+      ## ## Leaf-internal CO2concentration at standard conditions (Pa) gpp_st_io ## Gross primary
+      ## production (GPP) at standard conditions (kgC m-2 s-1) Top leaf nitrogen content per unit mass
+      ## (kgN kgLeaf-1); BETY: percent
```
Comment (mdietze) [inline models/jules/R/write.config.JULES.R]: here also

File: models/jules/R/write.config.JULES.R [hunk 4]
```diff
@@ -24,316 +24,335 @@
...
-      ## This is now replaced by a linear regression based on data reported in Kattge et al. 2009. Vint is the y-intercept, vsl is the slope.
-      ## vint_io     ## Units: μmol CO2 m-2 s-1.
-      ## vsl_io      ## Units: μmol CO2 gN-1 s-1
-      ## kn_io       ## Decay of nitrogen through the canopy
-      if(var == "leaf_respiration Q10") names(pft)[v] = "q10_leaf_io"
-
+      ## There is a linear relationship between Vcmax and Narea. Previously Vcmax was calculated as the
+      ## product of nl0 and neff.  This is now replaced by a linear regression based on data reported in
+      ## Kattge et al. 2009. Vint is the y-intercept, vsl is the slope.  vint_io ## Units: μmol CO2 m-2
+      ## s-1.  vsl_io ## Units: μmol CO2 gN-1 s-1 kn_io ## Decay of nitrogen through the canopy
```
Comment (mdietze) [inline models/jules/R/write.config.JULES.R]: here also

File: models/jules/R/write.config.JULES.R (hunk 1 above)
Comment (bpbond) [inline models/jules/R/write.config.JULES.R]: Whoops sorry for missing that.

Comment (bpbond) [inline models/jules/R/write.config.JULES.R]: Oh right...I didn't miss it, but remember wondering WTF was @mdietze doing here?
Bad tidy!

**Comment by mdietze** [comment]
Ready to go on Jules with Merge

**Comment by dlebauer** [comment]
@bpbond pecan has (thanks to @robkooper) logging functions and `logger.error` is generally  preferred to `stop`. There is also logger.info, logger.warn etc. I didn't look at the specific code here. I believe one exception is functions that are remotely executed (like those that read and convert model output) to reduce dependency issues.

**Comment by robkooper** [comment]
`logger.severe()` will show the message and quit, so you can do this instead of stop()

**Pull request merged**